libkbfs/identify_util.go

```diff
@@ -141,11 +141,18 @@ func getExtendedIdentify(ctx context.Context) (ei *extendedIdentify) {
 	}
 }
 
-func identifyUID(ctx context.Context, nug normalizedUsernameGetter, identifier identifier, uid keybase1.UID, isPublic bool) error {
+func identifyUID(ctx context.Context, nug normalizedUsernameGetter,
```

Comment:
Can you add a comment saying that `identifyUser` should be preferred unless only the UID is known, for example for rekeying?